bedrock/firefox/templates/firefox/dev-firstrun.html

```diff
@@ -147,7 +137,31 @@
             <div class="video" data-video-id="R_qDaLQ8ghg"></div>
           </div>
         </li>
-      {% endif %}
+        <li class="feature">
+          <a href="https://developer.mozilla.org/docs/Tools/Performance" rel="external">
+            <img src="{{ static('img/firefox/firstrun/dev/performance-tools.png') }}" alt="{{ _('Screenshot') }}" class="screenshot">
+          </a>
+          <h2>{{_('Performance Tools')}}</h2>
```

Comment:
Just verifying that this looks good. If `firefox-dev-fall` is off, I see it in the top featured section. If `firefox-dev-fall` is on, I see it below in the "Features and Tools" section. :+1: